docs/zh/samples/s3_configuration.md

```diff
@@ -0,0 +1,56 @@
+## 示例 - Fluid访问AWS S3所需的特殊配置
+
+如果选择AWS S3作为Alluxio的底层存储系统,Alluxio需要进行额外配置,以使得Alluxio能够正常访问挂载的S3存储系统.
+
+本文档展示了如何在Fluid中以声明式的方式完成Alluxio所需的特殊配置. 更多信息请参考[在Amazon AWS S3上配置Alluxio](https://docs.alluxio.io/os/user/stable/cn/ufs/S3.html)
+
+## 前提条件
+
+- [Fluid](https://github.com/fluid-cloudnative/fluid)
+- 配置完成的S3 bucket及有权限访问该bucket的AWS证书
+
+请参考[Fluid安装文档](https://github.com/fluid-cloudnative/fluid/blob/master/docs/zh/userguide/install.md)完成安装
+
+## 运行示例
+
+**创建Dataset资源对象**
+
+```yaml
+$ cat << EOF > dataset.yaml
+apiVersion: data.fluid.io/v1alpha1
+kind: Dataset
+metadata:
+  name: my-s3
+spec:
+  mounts:
+    - mountPoint: s3://<bucket-name>/<path-to-data>
+      name: s3
+      options:
+        aws.accessKeyId: XXXXXXXXXXXXX
+        aws.secretKey: XXXXXXXXXXXXXXXXXXXXXXX
```

Comment:
Thanks very much for your contributions! It would be great if  you are using `K8s secrets` instead of  plain text. Maybe it's like the following sample.

```
apiVersion: data.fluid.io/v1alpha1
kind: Dataset
metadata:
  name: hadoop
spec:
  mounts:
    - mountPoint: oss://<oss_bucket>/<bucket_dir>
      options:
        fs.oss.endpoint: <oss_endpoint>  
      name: hadoop
      encryptOptions:
        - name: aws.accessKeyId
          valueFrom:
            secretKeyRef:
              name: mysecret
              key: fs.oss.accessKeyId
        - name: aws.secretKey
          valueFrom:
            secretKeyRef:
              name: mysecret
              key: fs.oss.accessKeySecret
```